src/stepper/converter.ts

```diff
@@ -62,7 +62,7 @@ export function nodeToValueWithContext(node: substituterNodes, context: Context)
   return node.type === 'Literal'
     ? node.value
     : util.isBuiltinFunction(node)
-      ? builtin[(node as es.Identifier).name]
+      ? builtin[(node as es.Identifier).name as keyof typeof builtin]
```

Comment:
If we change `isBuiltInFunction` to a proper type guard we won't need the type assertion:

```ts
type BuiltinIdentifier = Omit<es.Identifier, 'name'> & {
  name: keyof typeof builtin
}

export function isBuiltinFunction(node: SubstituterNodes): node is BuiltinIdentifier {
  if (!isIdentifier(node) || !isKeyType(builtin, node.name)) return false
  // ...the other checks
}
```